scripts/build.sh

```diff
@@ -16,7 +16,7 @@ DISTDIR=dist
   mkdir -p $DISTDIR
   rm -rf $DISTDIR/*
 
- for d in opentelemetry-api/ opentelemetry-sdk/ opentelemetry-instrumentation/ opentelemetry-proto/ exporter/*/ instrumentation/*/ ; do
+ for d in opentelemetry-api/ opentelemetry-sdk/ opentelemetry-instrumentation/ opentelemetry-proto/ exporter/*/ instrumentation/*/ propagator/*/; do
```

Comment:
Nit: Should we name it `propagators` plural?

Reply:
eh..... maybe? though exporter is singular....

Reply:
+1 to leaving it as is, for consistencies sake with the other directories.